CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning].
 
 ## [Unreleased]
 
+- [PR-85](https://github.com/itk-dev/ai-screening/pull/85)
```

Comment:
Ohh, this belongs in another PR, but hopefully all will bemerged by the end of the day.